Formula/open-mpi.rb

```diff
@@ -53,7 +102,14 @@ def install
     args << "--enable-mpi-java" if build.with? "java"
     args << "--enable-mpi-cxx" if build.with? "cxx-bindings"
 
-    system "./autogen.pl" if build.head?
+    # Change the `autogen.pl` handling back to `system "./autogen.pl if build.head?` once the
+    # patches listed earlier are all in a stable Open MPI release:
+    if build.stable?
+      autogen_args = %w[--force]
```

Comment:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Additionally, I wonder if I could get rid of this hack and just do something with `autoreconf` for the `stable` spec instead, as that would likely be better than running `./autogen.pl --force`…_looks._

Reply:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Never mind; Open MPI's [`HACKING` guide](https://github.com/open-mpi/ompi/blob/master/HACKING) says, as I might have seen before, the following on [lines 206–210](https://github.com/open-mpi/ompi/blob/master/HACKING#L206-L210):  

```
⋮

   5a. You generally need to run autogen.pl whenever the top-level
       file "configure.ac" changes, or any files in the config/ or
       <project>/config/ directories change (these directories are
       where a lot of "include" files for OMPI's configure script
       live).

⋮
```

So, alas, no direct `autoreconf` for me.  (_Shrugs._)